sympy/physics/mechanics/functions.py

```diff
@@ -399,7 +399,7 @@ def Lagrangian(frame, *body):
     return kinetic_energy(frame, *body) - potential_energy(*body)
 
 
-def find_dynamicsymbols(expression, exclude=None):
+def find_dynamicsymbols(expression, exclude=None, reference_frame=None):
```

Comment:
Please include an explanation of this new argument in the docstring for this function. An example of use it helpful too.